pkgs/applications/editors/emacs/elisp-packages/manual-packages/elpaca/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  melpaBuild,
+  fetchFromGitHub,
+  git,
```

Comment:
It fails to build.  Please make sure this `git` is `pkgs.git` instead of `emacs.pkgs.git`.  Example: https://github.com/NixOS/nixpkgs/blob/4010f08a6b4a2f9a41193779aaec324fbecb3143/pkgs/applications/editors/emacs/elisp-packages/manual-packages.nix#L21

Reply:
This is not yet resolved.  Please also add an entry for `elpaca` in the `manual-packages.nix` file like the `lsp-bridge` example I gave.